analyzer/tools/build-logger/README.md

```diff
@@ -88,3 +88,10 @@ are provided to the compiler then the complete build action will be thrown
 away. This means that build actions which only perform linking will not be
 captured. We consider a file as object file if its extension is `.o`, `.so` or
 `.a`.
+
+### `CC_LOGGER_FALLBACK`
```

Comment:
Please make the old mode the default and allow to switch the new mode with an env var.
It would be too risky to release this change with so short testing period and testing on production code.

In CodeChecker 6.16.0 we can make this new mode the default if it works without any issues and also solves the probelms of the users.

Reply:
Updated accordingly.